test/test_helper.rb

```diff
@@ -15,6 +15,7 @@
 end
 
 require 'bundler'
```

Comment:
```suggestion
require 'bundler/setup'
```
can do the same thing

Reply:
Oh, thanks for the hint.